ansible/roles/ceph-collectd/tasks/configure_collectd.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: Ship collectd.conf
+  copy:
+    src: files/etc/collectd.conf
+    dest: /etc/
+  notify: Restart collectd
+
+- name: Ship /etc/collectd.d
+  copy:
+    src: files/etc/collectd.d
+    dest: /etc/
+  notify: Restart collectd
+
+- name: Set hostname in write_graphite.conf
+  replace:
+    dest: "/etc/collectd.d/write_graphite.conf"
+    regexp: 'Host ".*"'
+    replace: "Host \"{{ groups['ceph-grafana'][0] }}\""
+  notify: Restart collectd
+
+- name: Set cluster name in cephmetrics.conf
+  replace:
+    dest: /etc/collectd.d/cephmetrics.conf
+    regexp: 'ClusterName ".*"'
+    replace: 'ClusterName "{{ cluster }}"'
+  notify: Restart collectd
+
+- name: Set plugin path in cephmetrics.conf
+  replace:
+    dest: /etc/collectd.d/cephmetrics.conf
+    regexp: 'ModulePath ".*"'
+    replace: 'ModulePath "/usr/lib64/collectd/cephmetrics"'
```

Comment:
I think we could just unify this. I renamed the generic 'python-plugins' to 'cephmetrics' to avoid conflicts and the devel_mode should be fine doing that as well. You just need to change the dest in line 19 of  ansible/roles/ceph-collectd/tasks/install_collectd_plugins.yml to unify this (and the path in the cephmetrics.conf).